### PR TITLE
Firefox logo doesn't render in Chrome #35

### LINKED ISSUE #35: Firefox logo doesn't render in Chrome (for static pages)
blipz-n-chitz

### PR DESCRIPTION
**Pull request by wisniewskit**
I just ended up folding the Firefox logo SVGs into one, which fixes Chrome not being able to display it, and also simplifies the HTML (especially by removing those awkward linting comments).

I also drive-by changed the alt-text to "broken page" for the icon, to address [this comment](https://github.com/mozilla/webcompat-blipz-experiment/pull/30#pullrequestreview-118537236).

Peter, I'm not sure if the linter config change is the best solution, but I don't really like having a non-empty `alt` attributes for logos and other presentation-only `img` tags (who really wants to hear their screen reader pointlessly say "logo"?). But I'd also prefer leaving this rule on if it allowed `alt=""`, so if you know of a better way to do this please share!

Fixes #35

### CODE REVIEW

**Comment by miketaylr** [comment]
I'll defer review to @pdehaan. I did poke around mozilla.org, and I see the logo being used as a bg image for a containing parent in 2 places, or a bg image for a ::before pseudo element. Might be another option to avoid having to hack linting rules?

File: .htmllintrc [hunk 1]
```diff
@@ -3,5 +3,6 @@
   "head-req-title": null,
   "id-class-style": "camel",
   "indent-width": 2,
-  "indent-width-cont": true
+  "indent-width-cont": true,
+  "img-req-alt": false
```
Comment (pdehaan) [inline .htmllintrc]: Per https://github.com/htmllint/htmllint/wiki/Options#img-req-alt, lets change this to:

```json
"img-req-alt": "allownull"
```

That should allow an empty string, and still throw errors if we have an `` tag without an `alt=` attribute.

**Comment by wisniewskit** [comment]
>Might be another option to avoid having to hack linting rules?

Sounds like a hack to me either way, but maybe that's the better choice.

File: .htmllintrc (hunk 1 above)
Comment (wisniewskit) [reply .htmllintrc]: And you're ok with that?

File: docs/firefox-logo.svg [hunk 2]
```diff
@@ -1,3 +1,5 @@
+<svg id="logo" xmlns="http://www.w3.org/2000/svg" version="1.1" viewBox="0 0 100 100">
+<defs>
```
Comment (pdehaan) [inline docs/firefox-logo.svg]: Now that this is nested, the indenting looks a bit funny, but whatever... maybe I'll file a separate bug to run this through SVGO or something to see if we can shave a couple bytes from the file.

File: .htmllintrc (hunk 1 above)
Comment (wisniewskit) [reply .htmllintrc]: Ah, ok. I'll do that and merge the final result. Thanks!

File: docs/firefox-logo.svg (hunk 2 above)
Comment (wisniewskit) [reply docs/firefox-logo.svg]: Yeah, I can definitely fix that if it's enough of a nuisance to bother with :)

**Pull request merged**